tests/client/test_client_tenant_token.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=invalid-name
+
+from re import search
+import pytest
+import meilisearch
+from tests import BASE_URL, MASTER_KEY
+from meilisearch.errors import MeiliSearchApiError
+import datetime
+
+def test_generate_tenant_token_with_search_rules(get_private_key, index_with_documents):
+    """Tests create a tenant token with only search rules."""
+    index_with_documents()
```

Comment:
Did you have a setup handler without documents or a simple index creation? Because it's good to avoid unnecessarily setup :D

Reply:
I had documents to check the search return at least a few documents.